README.md

```diff
@@ -23,10 +28,12 @@ make check
 ## Usage
 
 ```
-kvm-host [bzImage]
+build/kvm-host -k [bzImage] -i [initrd]
```

Comment:
Denote the command line.
See https://stackoverflow.com/questions/21503865/how-to-denote-that-a-command-line-argument-is-optional-when-printing-usage